hibernate5/src/main/java/com/fasterxml/jackson/datatype/hibernate5/HibernateProxySerializer.java

```diff
@@ -1,24 +1,21 @@
 package com.fasterxml.jackson.datatype.hibernate5;
 
-import java.io.IOException;
-import java.util.HashMap;
-
 import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.BeanProperty;
-import com.fasterxml.jackson.databind.JavaType;
-import com.fasterxml.jackson.databind.JsonMappingException;
-import com.fasterxml.jackson.databind.JsonSerializer;
-import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.*;
 import com.fasterxml.jackson.databind.jsonFormatVisitors.JsonFormatVisitorWrapper;
 import com.fasterxml.jackson.databind.jsontype.TypeSerializer;
 import com.fasterxml.jackson.databind.ser.ContextualSerializer;
 import com.fasterxml.jackson.databind.ser.impl.PropertySerializerMap;
-
 import org.hibernate.engine.spi.Mapping;
-import org.hibernate.engine.spi.SessionImplementor;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
```

Comment:
@cowtowncoder  This [commit](https://github.com/hibernate/hibernate-orm/commit/774b805ce1bd01abde8f18c399ab49580b569263) introduced the `SessionImplementor.java`, apparently, that was introduced since `5.2.0`, as it is in the feature branch `5.2.0` 

I did not see any feature used in this PR for `SharedSessionContractImplementor ` class.